config_panel.toml

```diff
@@ -0,0 +1,198 @@
+        default = "127.0.0.1:8001"
+        help = "The path is always /metrics"
+        bind = "metrics>listen:__FINALPATH__/config.yaml"
+
+[whatsapp]
+name = "Whatsapp configuration"
+help = "Configuration for things that are directly sent to WhatsApp"
+services = ["__APP__"]
+
+    [whatsapp.config]
+    name = "General configuration"
+
+        [whatsapp.config.os_name]
+        ask = "Device name that's shown in the 'WhatsApp Web' section in the mobile app"
+        type = "string"
+        default = "Mautrix-WhatsApp bridge"
+        bind = "whatsapp>os_name:__FINALPATH__/config.yaml"
+
+        [whatsapp.config.browser_name]
+        ask = "Browser name that determines the logo shown in the mobile app"
+        type = "select"
+        choices = ["unknown", "chrome", "firefox", "ie", "opera", "safari", "edge", "desktop", "ipad", "android_tablet", "ohana", "aloha", "catalina", "tcl_tv"]
+        default = "unknown"
+        help = "Must be 'unknown' for a generic icon or a valid browser name if you want a specific icon"
+        bind = "whatsapp>browser_name:__FINALPATH__/config.yaml"
+
+[bridge]
+name = "Bridge configuration"
+services = ["__APP__"]
+
+    [bridge.config]
+    name = "General configuration"
+
+        [bridge.config.username_template]
+        ask = "Localpart template of MXIDs for WhatsApp users"
+        type = "string"
+        default = "whatsapp_{{.}}"
+        help = "{{.}} is replaced with the phone number of the WhatsApp user"
+        bind = "bridge>username_template:__FINALPATH__/config.yaml"
+
+[encryption]
+name = "Encryption"
+services = ["__APP__"]
+    [encryption.config]
+    name = "Encryption settings"
+
+        [encryption.config.encryption]
+        ask = "Enable encryption"
+        type = "boolean"
+        yes = "true"
+		no = "false"
+        help = "Enables end to bridge encryption"
+        bind = "encryption>allow:__FINALPATH__/config.yaml"
+
+        [encryption.config.default_encryption]
+        ask = "Default encryption"
+        type = "boolean"
+        yes = "true"
+        no = "false"
+        help = "Force-enable encryption in all portals the bridge creates. This will cause the bridge bot to be in private chats for the encryption to work properly."
+        bind = "encryption>default:__FINALPATH__/config.yaml"
+
+        [encryption.config.require_encryption]
+        ask = "Require encryption"
+        type = "boolean"
+        yes = "true"
+        no = "false"
+        help = "Require encryption, drop any unencrypted messages."
+        bind = "encryption>require:__FINALPATH__/config.yaml"
+
+[usermanagement]
+name = "User management"
+services = ["__APP__"]
+
+    [usermanagement.config]
+    name = "User management"
+
+        [usermanagement.config.helptext]
+        ask = "Allowed values:\n&emsp;&emsp;* - All Matrix users\n&emsp;&emsp;domain - All users on that homeserver\n&emsp;&emsp;mxid (@user:matrix.org) - Specific Matrix user\n&emsp;&emsp;username - Specific local user"
```

Comment:
Minor optional comment but I would encourage to maybe use a multi-line string here

Reply:
Hi @alexAubin 
Could you kindly tell me how to achieve that in toml? :)

Reply:
up @alexAubin

Reply:
Yes there are examples in https://toml.io/en/v1.0.0#string

Basically something like

```toml
ask = '''
Allowed values:
- * : All Matrix users
- domain : All users on that homeserver
- mxid (@user:matrix.org) : Specific Matrix user
- username : Specific local user
'''
```